src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4100,6 +4141,15 @@ namespace HVACVariableRefrigerantFlow {
                 }
             }
 
+            if (!lAlphaFieldBlanks(19)) {
```

Comment:
New IDD control zone name field in TU can be blank (set point controlled) or entered (load based control).